### ISSUE TITLE
meaning of ||f||

### ISSUE THREAD

**Issue by wjczf123** [deeplearning-wisc/npos#1]
Hi, leitian. 

This work is very interesting. 

I have a question about Eq. (8). What is the meaning of ||f||? Is the norm of predicted logits f(x)?

**Comment by taoleitian** [comment]
Correct, it is the norm of predicted logits. Thanks for your attention, maybe you can get more reference information from this paper here——https://arxiv.org/abs/2205.09310.

**Comment by wjczf123** [comment]
OK. Thanks.
